source/target/nordic/nrf5x/flash_blob.c

```diff
@@ -44,6 +44,14 @@ static const uint32_t nRF52832AA_FLM[] = {
     0x40010404, 0x40010504, 0x6e524635, 0x00000000, 
 };
 
+static const sector_info_t sectors_info[] = {
```

Comment:
Fyi, you don't need to add `sector_info_t` structures unless you have multiple sector sizes. The code in `target_flash_erase_sector_size()` will return `target_device.sector_size` if `target_device.sector_info_length` is 0. And that's the only use of this sector size info.

Reply:
target_device.sector_size and target_device.sector_cnt are removed if sector_info[] is added.

Reply:
Right, but you can simplify the code by keeping `sector_size` and `sector_cnt` and not having `sector_info`. Fewer changes, too.

(Btw, `sector_cnt` doesn't appear to be used anywhere in the code!)

Reply:
Since some targets have multiple sector sizes, would it be better to use the same data structure, sector_info?

Reply:
Generally yes, but it is already the case that most targets use `sector_size`. So unless we make a global change to `sector_info`, there will be a mix. Maybe we can do this when we switch to Pack-based algos. For now, I think it's best to limit changes unless required.